tpcx_bb/queries/q01/tpcx_bb_query_01_sql.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+
+from blazingsql import BlazingContext
+from xbb_tools.cluster_startup import attach_to_cluster
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
+import os
+
+from xbb_tools.utils import (
+    benchmark,
+    tpcxbb_argparser,
+    run_bsql_query,
+)
+
+cli_args = tpcxbb_argparser()
+
+
+@benchmark(
+    compute_result=cli_args["get_read_time"], dask_profile=cli_args["dask_profile"]
+)
+def read_tables(data_dir):
+    bc.create_table("item", data_dir + "/item/*.parquet")
+    bc.create_table("store_sales", data_dir + "/store_sales/*.parquet")
+
+
+@benchmark(dask_profile=cli_args["dask_profile"])
+def main(data_dir, client):
+    read_tables(data_dir)
+
+    query_distinct = """
+        SELECT DISTINCT ss_item_sk, ss_ticket_number
+        FROM store_sales s, item i
+        WHERE s.ss_item_sk = i.i_item_sk
+        AND i.i_category_id IN (1, 2 ,3)
+        AND s.ss_store_sk IN (10, 20, 33, 40, 50)
```

Comment:
It would be nice, but not critical, to factor out the desired category IDs and store secret keys into separate variables and then pass them to this query string. This is how the original Spark queries are implemented.
```
q01_i_category_id_IN = ...
q01_ss_store_sk_IN = ...
```

Reply:
Sure, I will apply the same kind of improvement about the query string parts as the original Spark version for all the other PRs.